build/README.md

```diff
@@ -41,21 +41,21 @@ The scripts directly under [`build/`](.) are used to build and test.  They will
 
 The `kube-build` container image is built by first creating a "context" directory in `_output/images/build-image`.  It is done there instead of at the root of the Kubernetes repo to minimize the amount of data we need to package up when building the image.
 
-There are 3 different containers instances that are run from this image.  The first is a "data" container to store all data that needs to persist across to support incremental builds. Next there is an "rsync" container that is used to transfer data in and out to the data container.  Lastly there is a "build" container that is used for actually doing build actions.  The data container persists across runs while the rsync and build containers are deleted after each use.
+There are 3 different containers instances that are run from this image.  The first is a "data" container to store all data that needs to persist across to support incremental builds. Next, there is a "rsync" container that is used to transfer data in and out of the data container.  Lastly, there is a "build" container that is used for actually doing build actions.  The data container persists across runs while the rsync and build containers are deleted after each use.
```

Comment:
```suggestion
There are 3 different containers instances that are run from this image.  The first is a "data" container to store all data that needs to persist across to support incremental builds. Next, there is an "rsync" container that is used to transfer data in and out of the data container.  Lastly, there is a "build" container that is used for actually doing build actions.  The data container persists across runs while the rsync and build containers are deleted after each use.
```

Reply:
The a/an rule is about how the word is pronounced, not how it is written -- "rsync" is pronounced "are-sync" so it gets an "an"